contributing-guides/style-guide.md

```diff
@@ -481,6 +477,18 @@ When documenting optional placeholders like paths or file extensions, it is sugg
 
 - Use `{{path/to/source.ext}}` instead of `{{path/to/source.tar[.gz|.bz2|.xz]}}`.
 
+### Keypress syntax
+
+To mark keypresses for TUI or UI programs, use angle brackets `<` and `>`.
+
+- Single characters example: `<a>`.
+- Special keys are to be written with pascal case: `<Ctrl>`, `<Super>`, `<Alt>`, `<Shift>`, `<Cmd>`, `<Option>`, `<Windows>`, `<Enter>`, `<Home>`, `<Space>`, `<Esc>`, `<ArrowUp>`, `<ArrowLeft>`, `<PageUp>`, ...
+- Special keys can be translated if they have culturally relevant translations.
+- When a program takes in uppercase character literals mark them as `<A>` instead of marking it with shift. Otherwise always mark characters in lowercase.
+- Mark simultaneous keypresses inside the same angle brackets separated by a single space:  `<Ctrl c>`, `<Alt F4>`, `<Ctrl Shift k>`, `<Super Shift PrtSc>`.
+- Consecutive keypresses need to be contained in their own angle brackets with no space in between: `<:><w><q><Enter>`, `<Ctrl k><Ctrl s>`.
+- Keypresses in the same context or when they form words can be grouped together in the same angle brackets: `<:><help><Enter>`.
```

Comment:
Question remain how to make sure that it remains clear that the grouped keypressed are individual keypresses.

Reply:
Maybe text that is typed out to a prompt, similar to a terminal can be excluded from being marked a keypress? `:` should still be marked as a keypress because it changes context.
```suggestion
- Keypresses in the same context or when they form words can be grouped together in the same angle brackets: `<:>help<Enter>`.
```